### ISSUE TITLE
Custom Object: Gesturerecognition doesnt seem to work

### ISSUE THREAD

**Issue by anoop4real** [olexale/arkit_flutter_plugin#38]
When I try to pan or scale on a custom object, the gesture recognizer handlers are not triggered nothing happens. I followed the box sample and instead of box sample I added a custom object to the plane.

**Comment by olexale** [comment]
Hi @anoop4real ,

Sorry for the late reply. Just to clarify: everything works fine with a primitive geometry (like a box), but doesn't work for a custom object loaded from, for example, dae file, right?

**Comment by anoop4real** [comment]
Yes, thats correct, I just used your Pokemon model and didnt work

**Comment by hacnguyen1412** [comment]
yeah, I have the same issue

**Comment by miftahulbagusp** [comment]
Yes, I have the same issue. Please fix.

**Comment by otmi100** [comment]
Can confirm this in 2024.. 

I was really excited to find this lib, but that's a downer :(
